timemachine/cpp/src/kernels/k_barostat.cuh

```diff
@@ -99,12 +100,13 @@ void __global__ k_setup_barostat_move(
 template <typename RealType>
 void __global__ k_decide_move(
     const int N,
+    const bool adaptive,
     const int num_molecules,
     const double kt,
     const double pressure,
     const RealType *__restrict__ rand, // [2] Use second value
     RealType *__restrict__ d_volume_delta,
-    RealType *__restrict__ d_volume_scale,
+    double *__restrict__ d_volume_scale,
```

Comment:
nit[type surprise]: should `d_volume_delta` and `d_volume_scale` share the same type?

Reply:
They don't have to, will switch d_volume_delta back

Reply:
Cchanged back in [2ebdfa5](https://github.com/proteneer/timemachine/pull/1152/commits/2ebdfa5b6309fefc61e599a00ebfe05af7af99a7)